### DISCUSSION TITLE
LDAP passwords default to SSHA

### DISCUSSION THREAD

**Discussion by simonwibberley** [authelia/authelia#6624] [Troubleshooting] [answered]
I'm attempting to configure Authelia to use LDAP user database, however the password algorithm defaults to SSHA. How do I set the default to, say, argon2id?

Thanks!

Simon

**Comment by simonwibberley** [comment]
Also, I am importing existing users. I can get OpenLDAP to recognise the agron2id hashed password via the phpldapadmin check password function, but authelia refuses the very same password...

**Comment by james-d-elliott** [comment, marked as answer]
Authelia doesn't care about the algorithm used since we just use a standard LDAP bind to check the password. As long as the LDAP backend is configured correctly it will work.
